tokio/src/runtime/builder.rs

```diff
@@ -1195,6 +1202,26 @@ cfg_time! {
             self.enable_time = true;
             self
         }
+
+        /// Sets auto-advance feature for paused timer.
+        ///
+        /// Read more at [crate::time::pause] doc.
+        ///
+        /// # Examples
+        ///
+        /// ```
+        /// use tokio::runtime;
+        ///
+        /// let rt = runtime::Builder::new_current_thread()
+        ///     .enable_time()
+        ///     .time_auto_advance(false)
+        ///     .build()
+        ///     .unwrap();
+        /// ```
+        pub fn time_auto_advance(&mut self, time_auto_advance: bool) -> &mut Self {
```

Comment:
Should this panic if using the multi-thread runtime flavor?

Reply:
Yeah, I think so, will add then